### DISCUSSION TITLE
Add preserveSpace to OrganizationChart for PrimeReact

### DISCUSSION THREAD

**Discussion by kov7si** [primefaces/community#3039] [PrimeReact]
Current behavior
Org Chart child nodes become hidden on collapse but occupy the same space making the org chart unnecessarily huge when all/some nodes are collapsed.

Expected behavior
The child nodes support `display:none` instead of `visibility:hidden`

Additional Information
This behaviour was already mentioned and solved in PrimeNG, it would be nice to have the same functionality in PrimeReact
[Link to discussion for PrimeNg](https://github.com/primefaces/primeng/issues/7313)
[Link to commit with fix](https://github.com/primefaces/primeng/commit/a4bc7aff7bd057b93451e5dea5d9a041b7eb17ce)

**Comment by xAl3xFx** [comment]
You can do this yourself. Just add this css rule to your index.css
```css
.p-organizationchart-nodes[style="visibility: hidden;"] {
    display: none;
}
